### PR TITLE
Fix ad badges conflict w/ player buttons

### PR DESCRIPTION
**Pull request by powerivq**
Move the ad badge to the bottom and the adchoice icon to the bottom right. Image attached.

### CODE REVIEW

**Review by calebcordry — COMMENTED** [review]
Thanks for taking this on, I think UX wanted to push this right if the button is there. Is that possible?

**Comment by powerivq** [comment]
> Thanks for taking this on, I think UX wanted to push this right if the button is there. Is that possible?

It is, but I feel it could make things  ambiguous: "Is this X button for closing the ad or story?" One might ask.

Screen Shot 2021-12-21 at 02 00 06

**Comment by calebcordry** [comment]
@hongcatlover WDYT?

**Comment by hongcatlover** [comment]
the X only shows with the Ad badge together if it is in a player environment, we are also trying to bring the page progress-bars back so the X and progress-bars would always showing consistently that would clear users confusions considering X as the access closing the Ad. + the upcoming auto advancing Ad behavior will help improving this too

**Comment by powerivq** [comment]
Screen Shot 2022-02-14 at 04 26 33

Updated format

File: extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-attribution.css [hunk 1]
```diff
@@ -1,9 +1,7 @@
-
-
 .i-amphtml-story-ad-attribution {
   position: absolute;
-  bottom: 0 !important;
-  left: 0 !important;
+  bottom: 10px !important;
+  right: 10px !important;
```
Comment (gmajoulet) [inline extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-attribution.css:4]: Just curious, you're not switching the badge to the other side of the page for RTL layouts?

Comment (powerivq) [reply extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-attribution.css:4]: It appears that we have not been doing that before (I don't know the right answer. Is it better or worse?). This change you pointed out is the adchoice icon.

**Pull request closed without merging**